docs/pages/getting-started/linux-server.mdx

```diff
@@ -70,22 +68,48 @@ Teleport (=teleport.version=) on Linux machines.
 
 Take a look at the [Installation Guide](../installation.mdx) for more options.
 
+(!docs/pages/includes/permission-warning.mdx!)
+
 ### Configure Teleport
 
 Generate a configuration file for Teleport using `teleport configure`.
 
-Acme turns on automatic TLS certificates from [Let's Encrypt](https://letsencrypt.org).
+<Tabs>
+  <TabItem label="Public internet eployment with Let's Encrypt">
+  Teleport uses Acme protocol to receive automatic TLS certificates from [Letsencrypt](https://letsencrypt.org).
 
-Set up an email to receive updates from Let's Encrypt, and use a valid DNS name for a cluster name.
+  First, set up an email to receive updates from Let's Encrypt. Use a valid DNS name for a cluster name.
 
-```code
-$ sudo teleport configure --acme --acme-email=your-email@example.com --cluster-name=tele.example.com -o file
-# Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
-```
+  ```code
+  $ sudo teleport configure --acme --acme-email=your-email@example.com --cluster-name=tele.example.com -o file
+  # Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
+  ```
+  </TabItem>
+
+  <TabItem label="Private net deployment">
+  Place the valid private key and a certificate chain in `/var/lib/teleport/privkey.pem`
+  and `/var/lib/teleport/fullchain.pem` respectively.
+
+  Configure teleport:
+
+  ```code
+  $ sudo teleport configure --cluster-name=tele.example.com -o file
+  $ sed -i '/^\s*https_keypairs/d' /etc/teleport.yaml
+  # Replace `tele.example.com` with your domain name.
+  $ export TELEPORT_PUBLIC_DNS_NAME="tele.example.com"
+  $ cat >> /etc/teleport.yaml <<EOF
+    public_addr: $TELEPORT_PUBLIC_DNS_NAME:443
+    web_listen_addr: 0.0.0.0:443
+    https_keypairs:
+    - key_file: /var/lib/teleport/privkey.pem
+      cert_file: /var/lib/teleport/fullchain.pem
+  EOF
```

Comment:
We could add a couple flag to `teleport configure` command to set public addr and key pairs, what do you think?

Then sed/cat wouldn't be needed and the command would be something like

```go
$ sudo teleport configure -o file \
    --cluster=tele.example.com \
    --public-addr=tele.example.com:443 \
    --key-file=/var/lib/teleport/privkey.pem \
    --cert-file=/var/lib/teleport/fullchain.pem
```

Let me know if you think it makes sense and I can create a ticket and we'll implement it before marketing release.

Reply:
@r0mant that would be nice, let's do it!

Reply:
@klizhentas - @greedy52 has implemented this and I've updated this PR: https://github.com/gravitational/teleport/pull/8988/commits/24d7b64420dd4aee93f99a5650c723593c1b28bc. Next 8.0 patch release will have these flags.